ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/StaticQueryExecutor.java

```diff
@@ -644,10 +645,16 @@ private static SourceName getSourceName(final Analysis analysis) {
     return source.getName();
   }
 
-  private static KsqlNode getOwner(final Struct rowKey, final Materialization mat) {
+  private static KsqlNode getOwner(
+      final KsqlConfig ksqlConfig,
+      final Struct rowKey,
+      final Materialization mat
+  ) {
     final Locator locator = mat.locator();
 
-    final long threshold = System.currentTimeMillis() + OWNERSHIP_TIMEOUT.toMillis();
+    final long timeoutMs =
+        ksqlConfig.getLong(KsqlConfig.KSQL_STREAMSTORE_REBALANCING_TIMEOUT_MS_CONFIG);
```

Comment:
Shouldn't this be `KSQL_PULL_QUERIES_OWNER_TIMEOUT_MS_CONFIG`?

Reply:
Yes, good catch.